utils/rpms/daos.spec

```diff
@@ -161,11 +141,7 @@ Requires: %{name}-client = %{version}-%{release}
 Requires: %{name} = %{version}-%{release}
 %endif
 Summary: The DAOS development libraries and headers
-%if %{defined cart_sha1}
-Requires: cart-devel-%{cart_sha1}
-%else
-Requires: cart-devel
-%endif
+Requires: cart-devel >= 4.6.1
```

Comment:
Since we have version in 2 places for Requires: cart and Requires: cart-devel , perhaps we should move version on top to avoid any mismatches in future?

Reply:
If we have to update the cart version again, we can do that.